src/features/player-history-aggregator/services/player.history.service.spec.js

```diff
@@ -1,42 +1,68 @@
-import { parsePlayerHistory } from "./player.history.service.js";
+import { addPlayerHistoriesFromSteamcharts } from "./player.history.service.js";
+import { Game } from "../../../models/game.js";
 import { eldenRingHttpDetailsSteamcharts } from "../../../../assets/steamcharts-details-pages/elden.ring.multiple.histories.html.details.page.js";
-import { sniperEliteHttpDetailsSteamcharts } from "../../../../assets/steamcharts-details-pages/sniper.elite.just.released.html.details.page.js";
+import { crushTheCastleHtmlDetailsSteamcharts } from "../../../../assets/steamcharts-details-pages/crush.the.castle.legacy.collection.html.details.page.js";
 
-describe("player.history.service.js", () => {
-  describe(".parsePlayerHistory", () => {
-    describe("if the game has multiple histories", () => {
-      let playerHistories;
+describe("player.history.service.js", function () {
+  describe(".addPlayerHistoriesFromSteamcharts adds the player histores from Steamcharts to each game object", function () {
+    describe("When both gamesPagesMap properties have proper values", function () {
+      beforeEach(function () {
+        this.map = new Map();
```

Comment:
No need for using of `this` where ever you create local scoped variables. If you're not using them for checks in `it` blocks use `const` or `let`. Update in the new code in this PR. Create an issue for the rest.

Reply:
Note for self: Continue here